login/pkg/constants/token.go

```diff
@@ -0,0 +1,8 @@
+package constants
+
+import "time"
+
+const (
+	APP_TOKEN_EXPIRE_TIME          = 3 * time.Hour
+	VERIFY_EMAIL_TOKEN_EXPIRE_TIME = 1 * time.Hour
```

Comment:
How about put here also the RESET_PASSWORD_TOKEN_EXPIRE_TIME constant from the forgot_password handler file?